server/validation/login.js

```diff
@@ -0,0 +1,9 @@
+const joi = require('@hapi/joi');
+
+module.exports = joi.object().keys({
+  secret: joi.string()
+    .invalid('<', '>')
+    .required(),
+  tableNumber: joi.number()
```

Comment:
Add a label for the `tableNumber` so the message displayed to the user is clearer
```suggestion
  tableNumber: joi.number().label('Table Number')
```